README.md

```diff
@@ -1,4 +1,6 @@
-# swift-snapshot-testing [![CircleCI](https://circleci.com/gh/pointfreeco/swift-snapshot-testing.svg?style=svg)](https://circleci.com/gh/pointfreeco/swift-snapshot-testing)
+# swift-snapshot-testing
+
+Darwin [![CircleCI](https://circleci.com/gh/pointfreeco/swift-snapshot-testing.svg?style=svg)](https://circleci.com/gh/pointfreeco/swift-snapshot-testing) Linux [![Build Status](https://travis-ci.org/pointfreeco/swift-snapshot-testing.svg?branch=master)](https://travis-ci.org/pointfreeco/swift-snapshot-testing)
```

Comment:
this is what it looks like:

<img width="297" alt="untitled" src="https://user-images.githubusercontent.com/135203/32700789-efc51ad8-c797-11e7-81d6-f53cc1327e32.png">

not happy with "Darwin"... not sure what to call it tho...

Reply:
macOS?

Reply:
Ideally we run separately for macOS, iOS, tvOS, etc.

Reply:
true true! will do `macOS` for now but we should add tests for all platforms soon and then we can say macOS/iOS/tvOS...